plugins-compat-tester-model/pom.xml

```diff
@@ -53,4 +53,17 @@
       <artifactId>jenkins-core</artifactId>
   	</dependency>
   </dependencies>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-deploy-plugin</artifactId>
+        <configuration>
+          <skip>true</skip>
```

Comment:
I figure the module structure of this repo is uninteresting to consumers—they only wish to use the CLI artifact.

Reply:
his is incorrect - anyone that wants to write hooks needs to consume the model.   May aw well just push everything and be done.